.travis.yml

```diff
@@ -15,4 +15,5 @@ before_install:
   - sudo apt-get install -qq nodejs
   - echo "deb http://download.mono-project.com/repo/debian wheezy/snapshots/3.12.1 main" | sudo tee /etc/apt/sources.list.d/mono-xamarin.list
   - sudo apt-get install -qq mono-complete
-  - curl -s https://storage.googleapis.com/golang/go1.6.2.linux-amd64.tar.gz | sudo tar -C /usr/local -xzf -
+  - eval "$(sudo gimme 1.6.2)"
+  - go version ; go env
```

Comment:
`gimme` just installs go. `go version ; go env` shows that go is installed plus details. The build would fail if go doesn't install correctly. gimme is maintained by travis -> it is quite unlikely to fail.